.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           name: Run Tests
           command: |
             source activate ${ENV_NAME}
-            pytest --junitxml=test-reports/junit.xml -v -s --current-env --nbval-lax mpas/*.ipynb
+            pytest --junitxml=test-reports/junit.xml -v -s --current-env --nbval-lax mpas/*.ipynb Bias-Correction/*.ipynb
```

Comment:
Adding `no_output_timeout` here may address the issue 


```yaml
command: |
    no_output_timeout: 30m
    source activate ${ENV_NAME}	            
```

Reply:
`no_output_timeout` has to go at the same indexing as the `command:` Trying it as:

- run:
          name: Run Tests
          command: |  
                 source activate ${ENV_NAME}
                 pytest --junitxml=test-reports/junit.xml -v -s --current-env ...
          no_output_timeout: 30m